FreeRTOS/corePKCS11
# Fix type used in LogDebug format string

Description
-----------
Fixes #185

Test Steps
-----------
Build with latest esp32c3 reference integration targeting the esp32c3 with debug logging enabled for corePKCS11.

Checklist:
----------



Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
